### PR TITLE
Updating the installation of Tomcat to 7.0.72 and Fedora Messaging Toolbox to 4.6.0

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Dear @yinlinchen ,

Please find a cleaned and restructured pull request for updating these dependencies.

Thank you as always.

### CODE REVIEW

**Comment by jrgriffiniii** [comment]
Apologies for the urgent update, but it seems more appropriate that I attempt a second commit in response to: http://tomcat.apache.org/tomcat-7.0-doc/changelog.html

File: config/fedora_camel_toolbox.script [hunk 1]
```diff
@@ -1,4 +1,4 @@
-feature:repo-add mvn:org.fcrepo.camel/toolbox-features/4.5.2/xml/features
+feature:repo-add mvn:org.fcrepo.camel/toolbox-features/4.6.0/xml/features
 feature:install fcrepo-indexing-solr
```
Comment (yinlinchen) [inline config/fedora_camel_toolbox.script:2]: feature:install fcrepo-service-activemq

Comment (yinlinchen) [reply config/fedora_camel_toolbox.script:2]: feature:install fcrepo-indexing-solr

File: 4.6.0/config/fedora_camel_toolbox.script [hunk 2]
```diff
@@ -1,4 +1,4 @@
-feature:repo-add mvn:org.fcrepo.camel/toolbox-features/4.5.2/xml/features
+feature:repo-add mvn:org.fcrepo.camel/toolbox-features/4.6.0/xml/features
```
Comment (yinlinchen) [inline 4.6.0/config/fedora_camel_toolbox.script:1]: feature:install fcrepo-service-activemq

**Review by yinlinchen — COMMENTED** [review]
In scripts/fedora_camel_toolbox.sh
add

```
sed -i 's|rest.host=localhost$|rest.host=0.0.0.0|' /opt/karaf/etc/org.fcrepo.camel.reindexing.cfg
```

before 

```
sed -i 's|fcrepo.authUsername=$|fcrepo.authUsername=fedoraAdmin|' /opt/karaf/etc/org.fcrepo.camel.reindexing.cfg
```

and use https://github.com/yinlinchen/fcrepo4-release-test-suite to verify it.

**Pull request closed without merging**